test/unit/pdf_form_template_test.rb

```diff
@@ -0,0 +1,16 @@
+require 'test_helper'
+
+class PdfFormMappingTest < ActiveSupport::TestCase
+  PDF_FIXTURE = './test/fixtures/pdfs/form_1.pdf'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.